util/network/src/lib.rs

```diff
@@ -54,6 +54,9 @@
 //! }
 //! ```
 
+//TODO: use Poll from mio
+#![allow(deprecated)]
```

Comment:
Ok, this doesn't work, it supresses only `use` itself and still warns when actually used
Disregard this remark, then